inst/htmljs/animint.js

```diff
@@ -1696,8 +1696,7 @@ var animint = function (to_select, json_file) {
 	  return d["clickSelects.variable"] + " " + d["clickSelects.value"];
 	};
       }
-      elements.text("")
-        .append("svg:title")
```

Comment:
What is `elements.text("")` doing here? Removing it seems to solve the problem...